project/api/forms.py

```diff
@@ -1,14 +1,17 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
 
+from builtins import object
 from django import forms
 from django.core.files.images import get_image_dimensions
 from django.contrib.auth.models import User
-from models import Account
+from .models import Account
 
 class UpdatePassword(forms.ModelForm):
     """
     Form for updating User Password
     """
-    class Meta:
+    class Meta(object):
```

Comment:
I believe django calls for the old-style class calls with just `Meta` instead of `Meta(object)`. Best to revert this particular change